projects/packages/videopress/src/client/block-editor/blocks/video/block.json

```diff
@@ -103,6 +103,10 @@
 			"type": "boolean",
 			"default": true
 		},
+		"displayEmbed": {
```

Comment:
I know this name comes indirectly from the response body shape, but `displayEmbed` doesn't sound very well to me, considering its purpose :-/